master/buildbot/test/fake/docker.py

```diff
@@ -15,8 +15,12 @@
 
 
 class Client(object):
+    latest = None
 
     def __init__(self, base_url):
+        Client.latest = self
+        self.call_args_create_container = []
```

Comment:
my usual question for this kind of data members: is `self.call_args_create_container` a part of the API? in other words, do we expect outside world to use these data members?  if not, then make them private (= prepend their names w/ `_`)